libs/@local/hash-graph-sdk/python/graph_sdk/entity.py

```diff
@@ -0,0 +1,88 @@
+"""Definition of high-level entities.
+
+Entities are instantiated objects that are stored in the graph,
+they can be created through the graph client.
+"""
+from typing import Generic, Self, TypeVar
+
+from graph_client.models import (
+    EntityId,
+    EntityLinkOrder,
+    EntityMetadata,
+    EntityRecordId,
+    EntityTemporalMetadata,
+    LinkData,
+    LinkOrder,
+    ProvenanceMetadata,
+)
+from graph_types.base import EntityType
+from pydantic import BaseModel
+
+T = TypeVar("T", bound=EntityType)
+
+
+class Link(BaseModel):
+    """Aggregated link information for an entity."""
+
+    left: EntityId | "Entity" | None = None
+    right: EntityId | "Entity" | None = None
+
+    left_to_right_order: LinkOrder | None = None
+    right_to_left_order: LinkOrder | None = None
```

Comment:
This should not be added as this is going to be removed in the Graph as well.

Reply:
Thought so, thanks for the flag!